containers/entitlement-pdp/handlers/getEntitlements.go

```diff
@@ -11,14 +11,19 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	ErrPayloadUnmarshal = Error("payload json unmarshal failure")
+	ErrPayloadInvalid   = Error("payload json invalid")
+)
+
 var tracer = otel.Tracer("handlers")
 
-// Entity attribute model
+// EntityAttribute model
 // @Description Represents a single entity attribute.
 type EntityAttribute struct {
-	//Attribute, in URI format, e.g.: "https://example.org/attr/Classification/value/COI"
+	// A ttribute, in URI format, e.g.: "https://example.org/attr/Classification/value/COI"
 	Attribute string `json:"attribute" example:"https://example.org/attr/OPA/value/AddedByOPA"`
-	//Optional display name for the attribute
+	// Optional display name for the attribute
 	DisplayName string `json:"displayName" example:"Added By OPA"`
```

Comment:
```suggestion
	DisplayName string `json:"displayName,omitempty" example:"Added By OPA"`
```

https://stackoverflow.com/a/34053855